src/bucket.ts

```diff
@@ -378,6 +378,197 @@ type MakeAllFilesPublicPrivateResponse = [File[]];
+ * //-
+ * bucket.iam.getPolicy().then(function(data) {
+ *   const policy = data[0];
+ *   const apiResponse = data[1];
+ * });
+ *
+ * @example <caption>include:samples/iam.js</caption>
+ * region_tag:storage_view_bucket_iam_members
+ * Example of retrieving a bucket's IAM policy:
+ *
+ * @example <caption>include:samples/iam.js</caption>
+ * region_tag:storage_add_bucket_iam_member
+ * Example of adding to a bucket's IAM policy:
+ *
+ * @example <caption>include:samples/iam.js</caption>
+ * region_tag:storage_remove_bucket_iam_member
+ * Example of removing from a bucket's IAM policy:
+ */
+/**
+ * Cloud Storage uses access control lists (ACLs) to manage object and
+ * bucket access. ACLs are the mechanism you use to share objects with other
+ * users and allow other users to access your buckets and objects.
+ *
+ * An ACL consists of one or more entries, where each entry grants permissions
+ * to an entity. Permissions define the actions that can be performed against
+ * an object or bucket (for example, `READ` or `WRITE`); the entity defines
+ * who the permission applies to (for example, a specific user or group of
+ * users).
+ *
+ * The `acl` object on a Bucket instance provides methods to get you a list of
+ * the ACLs defined on your bucket, as well as set, update, and delete them.
+ *
+ * Buckets also have
+ * [default
+ * ACLs](https://cloud.google.com/storage/docs/access-control/lists#default)
+ * for all created files. Default ACLs specify permissions that all new
+ * objects added to the bucket will inherit by default. You can add, delete,
+ * get, and update entities and permissions for these as well with
+ * {@link Bucket#acl.default}.
+ *
+ * @see [About Access Control Lists]{@link http://goo.gl/6qBBPO}
+ * @see [Default ACLs]{@link https://cloud.google.com/storage/docs/access-control/lists#default}
+ *
+ * @name Bucket#acl
+ * @mixes Acl
+ * @property {Acl} default Cloud Storage Buckets have
+ * [default
+ * ACLs](https://cloud.google.com/storage/docs/access-control/lists#default)
+ * for all created files. You can add, delete, get, and update entities and
+ * permissions for these as well. The method signatures and examples are all
+ * the same, after only prefixing the method call with `default`.
+ *
+ * @example
+ * const {Storage} = require('@google-cloud/storage');
+ * const storage = new Storage();
+ *
+ * //-
+ * // Make a bucket's contents publicly readable.
+ * //-
+ * const myBucket = storage.bucket('my-bucket');
+ *
+ * const options = {
+ *   entity: 'allUsers',
+ *   role: storage.acl.READER_ROLE
+ * };
+ *
+ * myBucket.acl.add(options, function(err, aclObject) {});
+ *
+ * //-
+ * // If the callback is omitted, we'll return a Promise.
+ * //-
+ * myBucket.acl.add(options).then(function(data) {
+ *   const aclObject = data[0];
+ *   const apiResponse = data[1];
+ * });
+ *
+ * @example <caption>include:samples/acl.js</caption>
+ * region_tag:storage_print_bucket_acl
+ * Example of printing a bucket's ACL:
```

Comment:
Should there be an example here?

Reply:
It looks weird, but here is how that actually renders: 
![image](https://user-images.githubusercontent.com/723048/101402553-e3f05280-38a1-11eb-98b7-fd2df898b6e1.png)